src/TimeSteppers/correct_immersed_tendencies.jl

```diff
@@ -0,0 +1,65 @@
+using Oceananigans.Grids: xnode, ynode, znode, Center, AbstractGrid
+
+"""
+    correct_immersed_tendencies!(model)
+    
+Correct the tendency terms to implement no-slip boundary conditions on an immersed boundary
+ without the contribution from the non-hydrostatic pressure. 
+Makes velocity vanish within the immersed surface.
+"""
+
+correct_immersed_tendencies!(model, Δt, γⁿ, ζⁿ) =
```

Comment:
I think the docstring has to be placed directly above the function (with no whitespace) in order to get registered by Documenter (though I'm not 100% sure). Not really important here since this is not a user-facing function, however.

Reply:
If we fix it now it definitely won't be an issue later!